Core/Aspects/Logging/LogAspect.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Castle.DynamicProxy;
+using WebNetSample.Core.CrossCuttingConcerns.Logging;
+
+namespace Core.Aspects.Logging;
+
+public class LogAspect : BaseLogAspect
+{
+    public LogAspect(Type loggerService) : base(loggerService)
+    {
+    }
+
+    protected override void OnBefore(IInvocation invocation)
+    {
+        _loggerServiceBase.Info(GetLogDetail(invocation));
+    }
+
+    protected override LogDetail GetLogDetail(IInvocation invocation)
```

Comment:
What is a difference here between [base](https://github.com/osmanaliyardim/WebNetSample/pull/5/files#diff-93fde3a2a74b208eccbe234f57aba8e853eb762d7216ae537bd39d98c3ef4a61R22) implementation and this one? I really cannot see any difference. If it is not exist, let's remove this override

Reply:
I think I just forgot :)